md/release-notes.md

```diff
@@ -42,37 +48,53 @@ For existing artifacts, nothing has changed: their IDs is a UUID. You can switch
 ### UI Designer artifacts last update date
 Last update date of all UI Designer artifacts is not updated with the use of the preview any more. Only when changes are made.
 
+### UI Designer logs separate from Engine logs
+In Bonita Studio **Help** menu, you can now find UI Designer logs, distinct from Engine logs, to better monitor UI Designer activities.
+
+### A responsive login page 
+The login page, used in Bonita Portal as well as in the Living Applications, is now responsive, so as to look at its best on a desktop, tablet or mobile environment.
+
 ### BDM meta-information for Portal Administrator
 Since installing or updating BDM requires BPM Services to be paused, such rights are only granted to the Technical User.
 However, since the Portal Administrator can now install and update the BDM Access Control file from the Administrator Portal, he can now get information on BDM status and last update date.
 
 ### New layout for the Administrator Portal "User details" page
 All details of a user in the organization are now diplayed on an AngularJS page of the Administrator Portal, bringing still more consistency to the Administrator Portal.
 
+### Bonita Studio status bar and coolbar
+To allow you to work in Bonita Studio while some work is done by Bonita Engine in the background, the Eclipse status bar is now displayed at the bottom right hand corner of the studio window. 
+To keep the same working space in the studio, we have set the default size of the coolbar at "small". If you prefer big icons, with labels, go to "Edit > Preferences > Appearance" and set the "Coolbar default size" to "Normal".
+
 ### Business objects relationships
 The default relationship type between business objects is now "agreggation". It was "composition" before, which led to some validation errors: a single object cannot be linked to several parents with a composition relationship, but it can with an aggregation relationship.
 
-### API error messages
-Improvements of messages in case of error on API calls have been done: explain the cause and give options for easier error recoveries.
-
 ### Studio status bar
 At the bottom right hand corner of Bonita Studio, the Eclipse status bas has been added to give information about the back-end activities, especially in case of operations on remote repositories. In order not to lose diagram designing space, icons of the coolbar are now small by default. If you prefer them large, go to the studio Preferences menu.
```

Comment:
duplicated § about **Studio status bar**... you can merge them

Reply:
yes.